File: Task-001/wordcount.scala
import scala.io.Source

def wordCount (substr: String): Int =
  wordsAll.sliding(substr.length).count(window => window == substr)

def main(args: Array[String]) {
  val wordsAll = Source.fromFile("Shakespeare.txt").getLines.mkString.replace("\n", "").replaceAll("""[\p{Punct}]""", " ").toLowerCase()

  val wordsSet: Set[String] = wordsAll.split(" ").toSet

  for (x <- wordsSet) {
    var n = wordCount(x)
    println(s"$x: $n")
  }
}

File: Task-001/wordcount.py
import string

with open("Shakespeare.txt", "r") as f:
	words_all = f.read().replace("\n", "").lower()

for ch in string.punctuation:
	words_all = words_all.replace(ch, " ")

words_set = words_all.split(" ")

for x in words_set:
	n = words_all.count(x)
	print("{}: {}".format(x,n))u newuser